### PR TITLE
SERENITY-3727-portfolio-wallet-sync-cronjob-in-org-services

### PR DESCRIPTION
**Pull request by swang-cc**

### CODE REVIEW

File: src/python/serenity_types/account/core.py [hunk 1]
```diff
@@ -95,6 +95,11 @@ class AccountUpdateRequest(AccountBaseRequest):
     """
     Input for the update request
     """
+
+    account_id: UUID
```
Comment (mtzavellas-cw) [inline src/python/serenity_types/account/core.py:99]: The account id is part of the path, there is no need to add it in the request object

**Review by mtzavellas-cw — CHANGES_REQUESTED** [review]

File: src/python/serenity_types/account/core.py (hunk 1 above)
Comment (swang-cc) [reply src/python/serenity_types/account/core.py:99]: Will change it. bwt. I followed the portfolio sdk.

Comment (swang-cc) [reply src/python/serenity_types/account/core.py:99]: More evidences that I am following Portfolio SDK api.

Screen Shot 2023-04-24 at 09 56 27
Screen Shot 2023-04-24 at 09 56 37

Comment (mtzavellas-cw) [reply src/python/serenity_types/account/core.py:99]: Yep, you're right, the SDK type is slightly different to make it more convenient to use.

**Comment by swang-cc** [comment]
renaming branch, raise a new pr. closed this one.

**Pull request closed without merging**